modules/layers/src/solid-polygon-layer/solid-polygon-layer-vertex.glsl.js

```diff
@@ -22,19 +22,25 @@ export default `\
 #define SHADER_NAME solid-polygon-layer-vertex-shader
 
 attribute vec2 vertexPositions;
-attribute float vertexValid;
-attribute vec3 positions;
-attribute vec2 positions64xyLow;
 
 #ifdef IS_SIDE_VERTEX
+attribute vec3 instancedPositions;
```

Comment:
Nit: We call these `instancePositions` (`instance` not `instanced`) in all other places that I am aware of. Best to follow a single convention across our layers.